kinova_driver/src/kinova_arm.cpp

```diff
@@ -869,8 +879,24 @@ void KinovaArm::publishFingerPosition(void)
   finger_position_publisher_.publish(fingers.constructFingersMsg());
 }
 
+void KinovaArm::OvisPublishJointPosition(void)
+{
+  KinovaAngles current_angles;
+  kinova_comm_.getJointAngles(current_angles);
+  ovis_msgs::OvisJointPosition ovis_joint_position_msg;
+  ovis_joint_position_msg.joint_positions[0] = current_angles.Actuator1;
+  ovis_joint_position_msg.joint_positions[1] = current_angles.Actuator2;
+  ovis_joint_position_msg.joint_positions[2] = current_angles.Actuator3;
+  ovis_joint_position_msg.joint_positions[3] = current_angles.Actuator4;
+  ovis_joint_position_msg.joint_positions[4] = current_angles.Actuator5;
+  ovis_joint_position_msg.joint_positions[5] = current_angles.Actuator6;
+
+  ovis_joint_position_publisher_.publish(ovis_joint_position_msg);
+}
+
 void KinovaArm::statusTimer(const ros::TimerEvent&)
 {
+  OvisPublishJointPosition();
```

Comment:
Coming in with a super nitpicky detail:
The other publish functions start with a lowercase. Do with that information what you will.